servers/src/main/java/tachyon/worker/block/evictor/GreedyEvictor.java

```diff
@@ -123,33 +93,41 @@ private EvictionPlan freeSpace(long availableBytes, BlockStoreLocation location)
     for (BlockMeta block : victimBlocks) {
       // TODO: should avoid calling getParentDir
       int fromTierAlias = block.getParentDir().getParentTier().getTierAlias();
-      List<StorageTierView> toTiers = mManagerView.getTierViewsBelow(fromTierAlias);
-      StorageDirView toDir = selectDirToTransferBlock(block, toTiers, pendingBytesInDir);
-      if (toDir == null) {
+      List<StorageTierView> toTiers = view.getTierViewsBelow(fromTierAlias);
+      StorageDirView destDir = selectDestDir(block, toTiers, pendingBytesInDir);
+      if (destDir == null) {
```

Comment:
Are we going with `dest` for destination instead of `dst`? I noticed the style is different between the worker code and the rest of the codebase.